.npmrc

```diff
@@ -1 +1 @@
-package-lock=false
+package-lock=true
```

Comment:
I understand that we don't push a package-lock on a module to ensure that the using modules can update to the latest package versions from the package.json.

Reply:
This was done to get around https://balena.zulipchat.com/#narrow/stream/348930-balena-io.2Fflowzone/topic/Setup-node.20cache

Reply:
I'll remove them from all projects I added now that you added https://github.com/product-os/flowzone/pull/498  :)